package/batocera/emulators/retroarch/libretro/libretro-mupen64plus-next/libretro-mupen64plus-next.mk

```diff
@@ -23,7 +23,7 @@ ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_RPI4),y)
 LIBRETRO_MUPEN64PLUS_NEXT_PLATFORM = rpi4_64
 LIBRETRO_MUPEN64PLUS_NEXT_EXTRA_ARGS = ARCH=aarch64 FORCE_GLES3=1
 
-else ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_RPI3),y)
+else ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_RPI3)$(BR2_PACKAGE_BATOCERA_TARGET_RPIZERO2),y)
```

Comment:
The built image of rpizero2 is 32bit and uses the videocore driver.
https://github.com/batocera-linux/batocera.linux/blob/master/configs/batocera-rpizero2_defconfig#L2
https://github.com/batocera-linux/batocera.linux/blob/master/configs/batocera-rpizero2_defconfig#L75

Reply:
The only reason I did this was because by default it was not compiling.

This isn't strictly necessary for this PR anyway so I'll just undo it.